cargo-crev/src/opts.rs

```diff
@@ -459,22 +436,10 @@ pub enum Command {
     )]
     Verify(Verify),
 
-    /// Review a crate
-    #[structopt(name = "review")]
+    /// Review a crate (code review, security advisory, flag issues)
+    #[structopt(name = "review", alias = "report", alias = "flag", alias = "advise")]
```

Comment:
Am I correct in understanding that this changes the behavior of "cargo crev flag" to be identical to the behavior of "cargo crev review"?  That seems potentially bad for anyone who's built up muscle memory who might skim over the unchanged "positive" rating of the resulting review, might be better to drop the alias entirely - so anyone with old muscle memory gets a reminder that they instead need to use "cargo crev review --issue" instead to get the old behavior.